src/main/java/orangehrm/OrangeMyInfoPage.java

```diff
@@ -0,0 +1,112 @@
+package orangehrm;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class OrangeMyInfoPage {
+    private final WebDriver driver;
+    private WebDriverWait editWait;
+    private WebDriverWait saveWait;
+
+    @FindBy(id = "btnSave")
+    private WebElement editBtn;
+
+    @FindBy(id = "personal_txtEmpFirstName")
+    private WebElement nameField;
+
+    @FindBy(id = "personal_txtEmpLastName")
+    private WebElement surnameField;
+
+    @FindBy(id = "personal_optGender_1")
+    private WebElement genderMaleRadioBtn;
+
+    @FindBy(id = "personal_cmbMarital")
+    private WebElement maritalStatus;
+
+    @FindBy(id = "personal_cmbNation")
+    private WebElement nationality;
+
+    @FindBy(id = "personal_DOB")
+    private WebElement dateOfBirthField;
+
+    @FindBy(id = "menu_admin_viewAdminModule")
+    private WebElement adminModule;
+
+    public OrangeMyInfoPage(WebDriver driver){
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+        editWait = new WebDriverWait(driver, 10);
+        saveWait = new WebDriverWait(driver, 10);
+    }
+
+    public OrangeMyInfoPage clickEdit(){
+        editWait.until(ExpectedConditions.visibilityOfElementLocated(By.id("btnSave")));
+        editBtn.click();
+        return this;
+    }
+
+    public OrangeMyInfoPage enterName(String name){
+        nameField.clear();
+        nameField.sendKeys(name);
+        return this;
+    }
+
+    public OrangeMyInfoPage enterSurname(String surname){
+        surnameField.clear();
+        surnameField.sendKeys(surname);
+        return this;
+    }
+
+    public OrangeMyInfoPage setGender(){
+        genderMaleRadioBtn.click();
+        return this;
+    }
+
+    public OrangeMyInfoPage setMartialStatus(){
+        Select maritalStatusSelect = new Select(maritalStatus);
+        maritalStatusSelect.selectByIndex(1);
+        return this;
+    }
+
+    public OrangeMyInfoPage setNationality(){
+        Select nationalitySelect = new Select(nationality);
+        nationalitySelect.selectByValue("185");
+        return this;
+    }
```

Comment:
I would recommend to passing data as parameter rather than hardcode inside of method body.

Reply:
fixed with adding classes for input data

Reply:
method  `setNationality()` still does not have parameter